patches/server/0403-Ensure-safe-gateway-teleport.patch

```diff
@@ -3,24 +3,18 @@ From: kickash32 <kickash32@gmail.com>
 Date: Fri, 15 May 2020 01:10:03 -0400
 Subject: [PATCH] Ensure safe gateway teleport
```

Comment:
This patch needs to be renamed to something more correct now. And just change the author too. Its really a complete patch replacement rather than an edit.